code/_onclick/hud/generic_dextrous.dm

```diff
@@ -0,0 +1,63 @@
+//Used for normal mobs that have hands.
+/datum/hud/dextrous/New(mob/living/owner, ui_style = 'icons/mob/screen_midnight.dmi')
+	..()
+	var/obj/screen/using
+	var/obj/screen/inventory/inv_box
+
+	using = new /obj/screen/drop()
+	using.icon = ui_style
+	using.screen_loc = ui_drone_drop
+	static_inventory += using
+
+	inv_box = new /obj/screen/inventory/hand()
+	inv_box.name = "right hand"
+	inv_box.icon = ui_style
+	inv_box.icon_state = "hand_r"
+	inv_box.screen_loc = ui_rhand
+	inv_box.slot_id = slot_r_hand
+	static_inventory += inv_box
+
+	inv_box = new /obj/screen/inventory/hand()
+	inv_box.name = "left hand"
+	inv_box.icon = ui_style
+	inv_box.icon_state = "hand_l"
+	inv_box.screen_loc = ui_lhand
+	inv_box.slot_id = slot_l_hand
+	static_inventory += inv_box
+
+	using = new /obj/screen/swap_hand()
+	using.icon = ui_style
+	using.icon_state = "swap_1_m"
+	using.screen_loc = ui_swaphand1
+	static_inventory += using
+
+	using = new /obj/screen/swap_hand()
+	using.icon = ui_style
+	using.icon_state = "swap_2"
+	using.screen_loc = ui_swaphand2
+	static_inventory += using
+
+	mymob.client.screen = list()
+
+	for(var/obj/screen/inventory/inv in (static_inventory + toggleable_inventory))
+		if(inv.slot_id)
+			inv.hud = src
+			inv_slots[inv.slot_id] = inv
+			inv.update_icon()
+
+/datum/hud/destrous/persistant_inventory_update()
```

Comment:
MIsspelled "dextrous".
"persistant" is how it is in the code and I can't change it.